### PR TITLE
fix on variable, function

### PR DESCRIPTION
**Pull request by phandyph**

### CODE REVIEW

File: movie/src/components/ActionContainer/ActionContainer.jsx [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
+/**
+ * {sort}
+ * The reason: {sort}: this function have already sorted.
+ * and get it to use here.
+ */
+import { sort } from "../../util/sort";
 import { useState } from "react";
 
-const ActionContainer = ({ movies, onSearch }) => {
+const ActionContainer = ({ movies, onSearch, onGet }) => {
```
Comment (sereybuth120) [inline movie/src/components/ActionContainer/ActionContainer.jsx:12]: is the `onGet` being use in here?

File: movie/src/components/Card/Card.jsx [hunk 2]
```diff
@@ -1,39 +1,37 @@
...
-                <Item className="description">
-                  <Item className="runTime">{movie.vote_average}</Item>
-                  <p className="bold">{movie.title}</p>
-                  <p>Date: {movie.release_date}</p>
-                  <p>Rate: {movie.vote_average}</p>
-                  <p>Popularity: {movie.popularity}</p>
-                </Item>
-              </Grid>
+    <Grid container spacing={3}>
+      {movies.map((movie, i) => {
```
Comment (sereybuth120) [inline movie/src/components/Card/Card.jsx:13]: Please remove `i` of we dont need it anymore

File: movie/src/components/Icons/DotIcon/DotIcon.css [hunk 3]
```diff
@@ -1,21 +1,3 @@
...
-  right: 2px;
-  position: absolute;
-}
-
-.more-detail:hover {
-  cursor: pointer;
-  background-color: rgb(151, 237, 237);
-}
-
 body:not(.modal-open){
```
Comment (sereybuth120) [inline movie/src/components/Icons/DotIcon/DotIcon.css:1]: its still `.modal-open`? why not `.modalOpen`

File: movie/src/components/Icons/DotIcon/DotIcon.jsx [hunk 4]
```diff
@@ -3,9 +3,9 @@ import IconButton from "@mui/material/IconButton";
 import MoreHorizIcon from "@mui/icons-material/MoreHoriz";
 import { Menu } from "@mui/material";
 import "./DotIcon.css";
-import DotPopup from "../../PopupCard/DotPopup/DotPopup";
+import DetailPopup from "../../PopupCard/DetailPopup/DetailPopup";
 
-export default function LongMenu() {
+const MoreHorizICon = () => {
```
Comment (sereybuth120) [inline movie/src/components/Icons/DotIcon/DotIcon.jsx:8]: 1. `....ICon` ---> `....Icon`
2. The component name should be the same as the file name

File: movie/src/components/Navbar/Navbar.jsx [hunk 5]
```diff
@@ -1,111 +1,57 @@
...
-          <ul className="logo-container">
-            <li className="logo">TMDB</li>
-            <li className="logo-color"></li>
-          </ul>
-
-          <li className="w3-dropdown-hover w3-mobile li-nav">
+        <ul className="leftNav">
+          <img
+            className="logo"
+            src="https://www.themoviedb.org/assets/2/v4/logos/v2/blue_short-8e7b30f73a4020692ccca9c88bafe5dcb6f8a62a4c6bc55cd9ba82bb2cd95f6c.svg"
```
Comment (sereybuth120) [inline movie/src/components/Navbar/Navbar.jsx:17]: please create a `const` for this img `src`

File: movie/src/components/PopupCard/BoxDisplayGenres/BoxDisplayGenres.css [hunk 6]
```diff
@@ -4,21 +4,21 @@
   max-width: 300px;
 }
 
-.labelUnClick {
+.disableGenre {
```
Comment (sereybuth120) [inline movie/src/components/PopupCard/BoxDisplayGenres/BoxDisplayGenres.css:7]: Could give a specific name than this? `disableGenreSection` || `disableGenreButton`....?

File: movie/src/util/sort.js [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+      break;
+
+    default:
+      list = list.sort((a, b) =>
+        a.popularity < b.popularity ? 1 : -1
+      );
+      break;
+  }
+}
+
```
Comment (sereybuth120) [inline movie/src/util/sort.js:52]: all white space downhere

File: movie/src/components/PopupCard/BoxDisplayGenres/BoxDisplayGenres.jsx [hunk 8]
```diff
@@ -5,19 +5,21 @@ import "./BoxDisplayGenres.css";
 import { useState } from "react";
 
 const BoxDisplayGenres = ({ genres, expanded, onClick }) => {
```
Comment (sereybuth120) [inline movie/src/components/PopupCard/BoxDisplayGenres/BoxDisplayGenres.jsx:7]: This one as well, the component name does not make much sense 😅

File: movie/src/components/FilterCard/FilterCard.jsx
Comment (sereybuth120) [inline movie/src/components/FilterCard/FilterCard.jsx:1]: This should should change from 
`movie/src/components/FilterCard/FilterCard.jsx`
👇 
`movie/src/components/FilterCard/index.jsx`

and please do the same for other file with the same structure

File: movie/src/components/ActionContainer/ActionContainer.jsx (hunk 1 above)
Comment (phandyph) [reply movie/src/components/ActionContainer/ActionContainer.jsx:12]: It just my example, it was deleted now.

File: movie/src/components/Icons/DotIcon/DotIcon.css (hunk 3 above)
Comment (phandyph) [reply movie/src/components/Icons/DotIcon/DotIcon.css:1]: It is not className bong, it is default class of open modal. It used to make default padding-right: 15px to padding-right:0px.

File: movie/src/components/Card/Index.jsx [hunk 9]
```diff
@@ -0,0 +1,38 @@
+import "./Card.css";
+import Grid from "@mui/material/Grid";
+import { Item } from "semantic-ui-react";
+import DetailIcon from "../DetailIcon/Index";
+const IMAGE_URL =
+  "https://images.unsplash.com/photo-1503023345310-bd7c1de61c7d?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwcm9maWxlLXBhZ2V8N3x8fGVufDB8fHx8&w=1000&q=80";
+
+// Here, I am using movies from json. Not movies from sort
+const Card = ({ movies }) => {
+  console.log("Movies: ", movies);
```
Comment (sereybuth120) [inline movie/src/components/Card/Index.jsx:10]: please remove this console.log

**Review by sereybuth120 — APPROVED** [review]

**Pull request merged**